### ISSUE TITLE
Chapter 1, 2 notes

### ISSUE THREAD

**Issue by Nikolaj-K** [udlbook/udlbook#79]
So here's just some notes on the side, for chapter 1 and 2, of the sort I'd make if someone asked me to skim their thesis - make of it what you might

* (1.1) is it "“learning” a model.". I think you use this less than "training a model" in the book (it's kinda weird in my east, but okay)
* () Likewise "fits mathematical models" or "fits parameters in a mathematical model"?
* (1.2.2 Latent variables) "This is because images are generated by physical processes" Do you mean "a physical process" and do you mean how the content comes about or is this saying something about the mechanics of recording? I assume the former?
* (1.2.2 Latent variables) The variables that "have a simple probability distribution by design". This makes it not fully clear, I think, whether the variables are hypothetical or explicit and accessible part of the programming
* (1.3 Reinforcement learning) I observe that there's no mention of quality metrics before 1.3
* (1.3.1) and which [ones] were irrelevant
* (2.2.2 or in the notes at the end) May or may not also reflect on the term "energy"
* (Chapter 2) "The model is just a mathematical equation; when the inputs are passed through this equation, it computes..." Since you speak of in- and outputs, maybe say the model just comprises mathematical equations defining a function?
* (2.2) "as closely as possible" Did you fix a type at this point? In a bivalent situation, it can also be _as commonly_ as possible here, no? (You end up going through a regression model, but I'm not sure if this is clear at this point.)
* (2.3) I'd still already write "minimize this loss function [over the training data]:"
* More anally, in the footnote one could write "the loss" or "the loss function value" (as opposed to the function).
* By a similar sentiment I ponder whether the "ground truth" y_i should be called an output, as is done below. What the semantic distinction between data input and output is also becomes confusing in (Generative vs. discriminative models:) - at least it's not clear given we've only encountered purely numerical motivations
* (2.2.2) "Since the best parameters minimize this expression, we call this a least-squares loss." The antecedent of this statement really applies to every loss function. I'd probably refer to the expression more explicitly. "since smaller summands help minimizing this expression, this is called" or something of the sort
* The texts above (2.3) and (2.6) (the regression example context) both want to define the term "loss function". I'd probably comment that this was previously said, when coming around to it the second time.
* (2.3 Summary) by a linear function, such as a straight line.

----

* () Is a "probability model" (5 times mentioned) the same as a "probabilistic model" (3 times)?
* 6.19 I'd write "If the [symmetric] Hessian matrix is positive definite"
